examples/image_classification_kornia.ipynb

```diff
@@ -0,0 +1,3856 @@
+{
```

Comment:
<p>Moving lines 8-19 to a markdown cell would increase readability:</p><p> </p><p>**Note:** You can also provide several splits:</p><p><code>dataset = load_dataset("imagefolder", data_files={"train": ["path/to/file1", "path/to/file2"], "test": ["path/to/file3", "path/to/file4"]})</code></p><p> </p><p>Or load a dataset from your local:</p><p><code>dataset = load_dataset("imagefolder", data_dir="path_to_folder")</code></p><p> </p><p>If you want to share your dataset publicly or collaborate with others privately, you can also push your dataset to the hub very easily (and reload afterwards using load_dataset)!</p><p><code>dataset.push_to_hub("nielsr/eurosat")</code></p><p><code>dataset.push_to_hub("nielsr/eurosat", private=True)</code></p><p> </p><p>** Datasets Hub: **</p><p>At Hugging Face, we host a variety of NLP, vision, speech and multi-modal datasets. Simply head over to our Hub and search for datasets. If you find a dataset you would like to use, you can simply pass in the repository name to load the dataset from the <a href="https://huggingface.co/datasets" target="_blank">hub</a>. As an example, let's load the <a href="https://huggingface.co/datasets/cifar10" target="_blank">CIFAR-10 dataset</a> from the hub.</p><p><code>dataset = load_dataset("cifar10")</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/226/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>